src/data/decks.asm

```diff
@@ -1,4 +1,6 @@
-DeckPointers: ; 30000 (c:4000)
+DeckPointers:
+	deck_list_start
```

Comment:
This `deck_list_start` doesn't need to be at the start of the pointer table, right?